File: examples/happy_birthday.cs
// C# program to print Happy Birthday 
using System; 
	
class GFG 
{ 
	// Print first row 
	public static void Main() 
	{ 
		char ch = '@'; 
		for(int i=1; i<=34; i++) 
		{ 
			if (i==5||i==7||i==10||i==11||i==14||i==15|| i==16||i==18||i==19||i==20||i==22||i==24) 
				Console.Write(ch); 
			else Console.Write(" "); 
		} 
		// Print second row 
		Console.Write("\n"); 
		for(int i=1; i<=34; i++) 
		{ 
			if(i==5||i==7||i==9||i==12||i==14||i==16 ||i==18||i==20||i==22||i==24) 
				Console.Write(ch); 
			else Console.Write(" "); 
		} 
		// Print third row 
		Console.Write("\n"); 
		for (int i=1; i<=34; i++) 
		{ 
			if (i==5||i==6||i==7||i==9||i==10||i==11||i==12|| i==14||i==15||i==16||i==18||i==19||i==20|| i==22||i==23||i==24) 
				Console.Write(ch); 
			else Console.Write(" "); 
		} 
		// Print fourth row 
		Console.Write("\n"); 
		for (int i=1; i<=34; i++) 
		{ 
			if (i==5||i==7||i==9||i==12||i==14||i==18||i==23) 
				Console.Write(ch) ; 
			else Console.Write(" ") ; 
		
		} 
		
		// Print fifth row 
		Console.Write("\n"); 
		for (int i=1; i<=34; i++) 
		{ 
			if (i==5||i==7||i==9||i==12||i==14||i==18||i==23) 
				Console.Write(ch) ; 
			else Console.Write(" ");
		} 
		// Happy is printed, now print birthday row by row 
		Console.Write("\n"); 
		Console.Write("\n"); 
		Console.Write("\n"); 
		for (int i=1; i<=34; i++) 
		{ 
			if (i==2||i==3||i==4||i==6||i==7||i==8||i==10|| i==11||i==14||i==15||i==16||i==18||i==20|| i==22||i==23||i==27||i==28||i==31||i==33) 
				Console.Write(ch); 
			else Console.Write(" "); 
		} 
		Console.Write("\n"); 
		for(int i=1; i<=34; i++) 
		{ 
			if (i==2||i==4||i==7||i==10||i==12|| i==15||i==18||i==20||i==22||i==24|| i==26||i==29||i==31||i==33) 
				Console.Write(ch); 
			else Console.Write(" "); 
		} 
		Console.Write("\n"); 
		for (int i=1; i<=34; i++) 
		{ 
			if (i==2||i==3||i==4||i==7||i==10||i==11|| i==15||i==18||i==19||i==20||i==22|| i==24||i==26||i==27||i==28||i==29||i==31 ||i==32||i==33) 
				Console.Write(ch); 
			else Console.Write(" "); 
		} 
		Console.Write("\n"); 
		for (int i=1; i<=34; i++) 
		{ 
			if (i==2||i==4||i==7||i==10||i==12|| i==15||i==18||i==20||i==22|| i==24||i==26||i==29||i==32) 
				Console.Write(ch); 
			else Console.Write(" "); 
		} 
		Console.Write("\n"); 
		for (int i=1; i<=34; i++) 
		{ 
			if (i==2||i==3||i==4||i==6||i==7||i==8|| i==10||i==12||i==15||i==18||i==20|| i==23||i==22||i==26||i==29||i==32) 
				Console.Write(ch); 
			else Console.Write(" "); 
		}

File: examples/Happy_Birthday.py
import turtle
import random

# sets background
bg = turtle.Screen()
bg.bgcolor("black")

# Bottom Line 1
turtle.penup()
turtle.goto(-170,-180)
turtle.color("white")
turtle.pendown()
turtle.forward(350)

# Mid Line 2
turtle.penup()
turtle.goto(-160,-150)
turtle.color("white")
turtle.pendown()
turtle.forward(300)

# First Line 3
turtle.penup()
turtle.goto(-150,-120)
turtle.color("white")
turtle.pendown()
turtle.forward(250)

# Cake
turtle.penup()
turtle.goto(-100,-100)
turtle.color("white")
turtle.begin_fill()
turtle.pendown()
turtle.forward(140)
turtle.left(90)
turtle.forward(95)
turtle.left(90)
turtle.forward(140)
turtle.left(90)
turtle.forward(95)
turtle.end_fill()

# Candles
turtle.penup()
turtle.goto(-90,0)
turtle.color("red")
turtle.left(180)
turtle.pendown()
turtle.forward(20)

turtle.penup()
turtle.goto(-60,0)
turtle.color("blue")
turtle.pendown()
turtle.forward(20)

turtle.penup()
turtle.goto(-30,0)
turtle.color("yellow")
turtle.pendown()
turtle.forward(20)

turtle.penup()
turtle.goto(0,0)
turtle.color("green")
turtle.pendown()
turtle.forward(20)

turtle.penup()
turtle.goto(30,0)
turtle.color("purple")
turtle.pendown()
turtle.forward(20)

# Decoration
colors = ["red", "orange", "yellow", "green", "blue", "purple", "black"]
turtle.penup()
turtle.goto(-40,-50)
turtle.pendown()

for each_color in colors:
    angle = 360 / len(colors)
    turtle.color(each_color)
    turtle.circle(10)
    turtle.right(angle)
    turtle.forward(10)

# Happy Birthday message
turtle.penup()
turtle.goto(-150, 50)
turtle.color("pink")
turtle.pendown()
turtle.write("Happy Birthday To You!", None, None, "25pt bold")
turtle.color("black")
